repository/expression.js

```diff
@@ -0,0 +1,64 @@
+import { Expression, User, UserExpression } from '../models';
+
+class ExpressionRepository {
+  constructor(expression, user, userExpression) {
+    this._expression = expression;
```

Comment:
You dont need to hold all of this in the instance since you dont modify then at all. You can directly use what you import. This just creates confusion.